templates/wordpress-vanilla/files/.gitignore

```diff
@@ -0,0 +1,13 @@
+# Your config, not a part of WP upstream
+wordpress/wp-config.php
+
+# Custom plugins
+# wordpress/wp-content/plugins/classic-editor
+# wordpress/wp-content/plugins/woocommerce
```

Comment:
Why are these commented out ones here?

Reply:
Also, in vanilla, shouldn't they be committed as normal rather than any fancy download and copy system?

Reply:
So my thinking here is
- commit WP to `wordpress`
- commit themes and plugins to subdirs outside of `wordpress`, same way we're recommending using non-Composer plugins in the [deploy guide](https://docs.platform.sh/guides/wordpress/deploy/next-steps.html#adding-plugins-and-themes-without-composer)
- at build everythings moves into `wordpress` to run like you'd expect, but then you can update in each separate subdir or update as submodule however you'd like there.
- the gitignore lines are for when running Lando and the build hook is run, left as only an example but I'll remove.

Reply:
If you're doing vanilla WP, then you have no local build process.  As setup here, it won't work in local development because all of the modules are in the wrong place.

Vanilla WP should be, well, vanilla.

Reply:
I guess in this case I'm pushing local Lando, so users can keep all of it up to date the easiest way - whether thats downloading new versions straight away or updating submodules. Makes sense to me since we're recommending it that way already with the deploy guide above